tests/Constraints/XMLMatchesXSDConstraint.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Constraints;
+
+/**
+ * @internal
+ */
+final class XMLMatchesXSDConstraint extends \PHPUnit_Framework_Constraint
```

Comment:
is it copy-paste from gecko ?

Reply:
yeah it is

Reply:
then I'm not confident, put you as author at least ;P
maybe, let it be raised for official phpunit package ?